src/main/webapp/app/components/Tabs/tabs.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../variables.scss';
+
+.tab {
```

Comment:
Try to use module.scss instead for component with a very generic name, so the compiler will create a unique classname to prevent styling pollution